lib/uplinkc/convert.go

```diff
@@ -14,26 +14,40 @@ func newBucketInfo(bucket *storj.Bucket) C.BucketInfo {
 	return C.BucketInfo{
 		name:         C.CString(bucket.Name),
 		created:      C.int64_t(bucket.Created.Unix()),
-		path_cipher:  C.uint8_t(bucket.PathCipher),
+		path_cipher:  C.Cipher(bucket.PathCipher),
```

Comment:
nit: Ah, this kind of cast worries me, I would rather have a func that does an explicit conversion. It seems would be too easy to cast a `storj.Cipher` to `C.Cipher`.